tools/tests/package-tests.js

```diff
@@ -742,7 +742,7 @@ selftest.define("package skeleton creates correct versionsFrom", function () {
   var s = new Sandbox({ warehouse: { v1: { recommended: true } } });
   var token = utils.randomToken();
   var fullPackageName = "test:" + token;
-  var fsPackageName = "test_" + token;
+  var fsPackageName = token;
```

Comment:
It looks like the test create-publish-and-search in publish.js also runs `create --package` and makes assumptions about fsPackageName.